components/Credential.yml

```diff
@@ -23,7 +23,7 @@ components:
             type: string
         "id":
           type: string
-          description: The ID of the credential.
+          description: The ID of the credential. This property MAY be empty. The issuer SHOULD NOT auto-generate the id property, as this can lead to unrecoverable partitioning errors.
```

Comment:
```suggestion
          description: The ID of the credential. This property MAY be empty. The issuer SHOULD NOT auto-generate the id property since the client wanted to issue a VC without the id.
```